### PR TITLE
Improve flows of member filters, write new queries to support them.

### PR DESCRIPTION
**Pull request by nanderv**

### CODE REVIEW

File: datamining/procedures/filter_members.py [hunk 1]
```diff
@@ -13,24 +23,40 @@ def filter_members(found_committees, found_privacy_settings, on, include_honorar
...
-        query = query & Q(committees__pk__in=committees)
+    query = membership_period_filter & Q(is_anonymous_user=False)
 
     members = Member.objects.filter(query).distinct()
     return members
+
+
+def filter_members_by_privacy_option(option: str):
+    if option == "activities":
+        print("ACT")
```
Comment (nanderv) [inline datamining/procedures/filter_members.py:34]: should still remove print statements here.

File: datamining/procedures/filter_members.py [hunk 2]
```diff
@@ -13,24 +23,40 @@ def filter_members(found_committees, found_privacy_settings, on, include_honorar
...
+        mmbers = Member.objects.filter(privacy_publications=True, is_anonymous_user=False).all()
+        members = []
+        for member in mmbers:
+            # TODO: something with last year?
+            if member.is_currently_member():
+                members.append(member)
+        return members
+    if option =="reunions":
+        return Member.objects.filter(privacy_reunions=True, is_anonymous_user=False).all()
+    print("ERROR", option)
```
Comment (RemcoSwenker) [inline datamining/procedures/filter_members.py:52]: And print statement here

File: datamining/templates/datamining/member_filtering_group.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{% extends 'datamining/member_filtering_base.html' %}
+{% load get_list %}
+
+{% block form %}
+        <form action="#" method="get">
+
+    Please use standard iso date formats (i.e. 2020-12-31):
```
Comment (RemcoSwenker) [inline datamining/templates/datamining/member_filtering_group.html:7]: There is no date in this tab

File: datamining/templates/datamining/member_filtering_group.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+                            <option {% if committee.pk|to_string in list %}selected{% endif %}
+                                    value="{{ committee.pk }}">{{ committee.name }}</option>
+
+                        {% endfor %}
+                    </select>
+                {% endwith %}
+                </td>
+            </tr>
+{#            <tr>#}
```
Comment (RemcoSwenker) [inline datamining/templates/datamining/member_filtering_group.html:24]: Why is this part of privacy settings still here?

File: datamining/procedures/filter_members.py (hunk 2 above)
Comment (nanderv) [reply datamining/procedures/filter_members.py]: Replaced by logging statement.

**Review by RemcoSwenker — APPROVED** [review]
Looks good to me and looks like it works.

**Pull request merged**